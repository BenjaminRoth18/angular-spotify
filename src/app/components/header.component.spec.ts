import { TestBed, ComponentFixture, async } from '@angular/core/testing';
import { HeaderComponent } from './header.component';

describe('HeaderComponent', () => {
    let component: HeaderComponent;
    let fixture: ComponentFixture<HeaderComponent>;

    beforeEach(async(() => {
        TestBed.configureTestingModule({ declarations: [HeaderComponent] });

        fixture = TestBed.createComponent(HeaderComponent);
        component = fixture.debugElement.componentInstance;
    }));

    it('should create component', () => {
        expect(component).toBeTruthy();
    });

    describe('snaphots', () => {
        it('should create snapshot', () => {
            expect(fixture).toMatchSnapshot();
        });
    });
});
