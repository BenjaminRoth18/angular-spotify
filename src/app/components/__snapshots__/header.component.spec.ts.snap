// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderComponent snaphots should create snapshot 1`] = `
<sp-header>
  <nav
    class="navbar navbar-dark mb-5"
  >
    <a
      class="navbar-brand"
      routerlink="playlist"
    >
      My Spotify playlist!
    </a>
  </nav>
</sp-header>
`;
